authors/apps/authentication/serializers.py

```diff
@@ -110,11 +111,6 @@ def validate(self, data):
         # purpose of this flag to tell us whether the user has been banned
         # or otherwise deactivated. This will almost never be the case, but
         # it is worth checking for. Raise an exception in this case.
-        if not user.is_active:
```

Comment:
We will need this flag for users who want to delete their accounts.

Reply:
I have returned back the flag, this will be used hand in hand with the 'is_verified' method you had spoken about earlier.